plugins/modules/vmware_drs_override.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = '''
+---
+module: vmware_drs_override
```

Comment:
```suggestion
module: vmware_drs_override
version_added: '5.2.0'
```

Something that I overlook too often, but we should document when the module has been added.